keyboards/handwired/dactyl_coolio/keymaps/sam-id-b3/keymap.c

```diff
@@ -0,0 +1,85 @@
+#include QMK_KEYBOARD_H
+#include "sam-id-b3.h"
+
+// Enumerate the tap dances
+enum taps {
+	//TD_PRNS = 0,
+	//TD_BRCS, //Brackets
+	TD_EXLM = 0, //! and ?
```

Comment:
No need to specify zero.  Enums start at 0. 
```suggestion
	TD_EXLM, //! and ?
```